plugins/modules/ec2_vol.py

```diff
@@ -223,153 +228,165 @@
     }
 '''
 
-import time
+from ..module_utils.core import AnsibleAWSModule
+from ..module_utils.ec2 import camel_dict_to_snake_dict
+from ..module_utils.ec2 import boto3_tag_list_to_ansible_dict
+from ..module_utils.ec2 import ansible_dict_to_boto3_filter_list
+from ..module_utils.ec2 import ansible_dict_to_boto3_tag_list
+from ..module_utils.ec2 import compare_aws_tags
+from ..module_utils.ec2 import AWSRetry
+from ..module_utils.core import is_boto3_error_code
 
-from distutils.version import LooseVersion
 
 try:
-    import boto
-    import boto.ec2
-    import boto.exception
-    from boto.exception import BotoServerError
-    from boto.ec2.blockdevicemapping import BlockDeviceType
-    from boto.ec2.blockdevicemapping import BlockDeviceMapping
+    import botocore
 except ImportError:
     pass  # Taken care of by ec2.HAS_BOTO
```

Comment:
Niggle: this is Handled by AnsibleAWSModule rather than ec2.HAS_BOTO

Reply:
@tremble Added clarification on the imports.